examples/pod_api.rs

```diff
@@ -10,7 +10,7 @@ use kube::{
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    std::env::set_var("RUST_LOG", "info,kube=debug");
+    std::env::set_var("RUST_LOG", "info,kube=debug,tower_http=debug");
```

Comment:
Run this example to see the default outputs from `TraceLayer` which replaced `LogRequest`.
It's fully customizable as [documented](https://docs.rs/tower-http/0.1.0/tower_http/trace/index.html), but kept it simple.